packages/compass-aggregations/src/typings.d.ts

```diff
@@ -5,3 +5,4 @@ declare module 'mongodb-query-parser' {
     const isCollationValid: (collationString: string) => false | null | CollationOptions;
     export { isCollationValid };
 }
+declare module 'ejson-shell-parser';
```

Comment:
I would strongly consider fixing these types part of the work. [This package](https://github.com/mongodb-js/ejson-shell-parser/blob/master/package.json) has types, it just doesn't export them properly, would you be able to address this? At least we need a ticket to address this later but in the same project

Reply:
I'll create a follow up and do that next as it would require publishing of the package.

Reply:
created: [COMPASS-6204](https://jira.mongodb.org/browse/COMPASS-6204)

Reply:
You don't have publishing rights for the package?